peps/pep-0788.rst

```diff
@@ -655,10 +661,10 @@ Asynchronous callback example
 *****************************
 
 As stated in the Motivation_, there are many cases where it's desirable
-to call Python in an asynchronous callback, such as a signal handler. In that
-case, it's not safe to call :c:func:`PyInterpreterState_Hold`, because it's
-not guaranteed that :c:func:`PyThreadState_Ensure` will ever be called, which
-would deadlock finalization.
+to call Python in an asynchronous callback. In that case, it's not safe to
```

Comment:
```suggestion
to call Python in an asynchronous callback. In such cases, it's not safe to
```

You refer to multiple, plural "cases" in the previous sentence, so it seems clearer and less potentially confusing to match that here.